reaktive/src/commonMain/kotlin/com/badoo/reaktive/observable/Repeat.kt

```diff
@@ -4,18 +4,19 @@ import com.badoo.reaktive.base.ErrorCallback
 import com.badoo.reaktive.base.ValueCallback
 import com.badoo.reaktive.disposable.Disposable
 import com.badoo.reaktive.utils.atomic.AtomicInt
+import com.badoo.reaktive.utils.atomic.AtomicLong
 import com.badoo.reaktive.utils.serializer.serializer
 
 /**
- * Returns an [Observable] that automatically resubscribes to this [Observable] at most [count] times.
+ * Returns an [Observable] that automatically resubscribes to this [Observable] at most [times] times.
  *
  * Please refer to the corresponding RxJava [document](http://reactivex.io/RxJava/javadoc/io/reactivex/Observable.html#repeat-long-).
  */
-fun <T> Observable<T>.repeat(count: Int = -1): Observable<T> =
+fun <T> Observable<T>.repeat(times: Long = Long.MAX_VALUE): Observable<T> =
```

Comment:
Let's add the following check before returning the observable:
```kotlin
require(times >= 0L) { "Number of times must not be negative" }
```
RxJava does the same, so we need consistency here as well.

Reply:
Done